### PR TITLE
Implemented task for search request to google.com

### PR DESCRIPTION
**Pull request by vvaryshchuk**

### CODE REVIEW

File: vasyl_varyshchuk/Week_3/google_search_request.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+"""
+This module sends a request to google.com returning the response headers
+and the specified number of searching results
+"""
+import requests
+from bs4 import BeautifulSoup
```
Comment (pivancy) [inline vasyl_varyshchuk/Week_3/google_search_request.py]: Is the `bs4` a default library?

Comment (vvaryshchuk) [reply vasyl_varyshchuk/Week_3/google_search_request.py]: No, it isn't, Should I use only standard libraries?

Comment (pivancy) [reply vasyl_varyshchuk/Week_3/google_search_request.py]: You should:
1) add some message
2) add a handler for import non-default libraries
3) Add an `requirements.txt` file with a needed dependency(read more about this part)

Comment (vvaryshchuk) [reply vasyl_varyshchuk/Week_3/google_search_request.py]: I added an ability to install the required module from requirements.txt file and committed my changes.

File: vasyl_varyshchuk/Week_3/google_search_request.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    required_modules = parse_requirements('requirements.txt', session=False)
+    requirements = [str(i.req) for i in required_modules]
+    for requirement in requirements:
+        main(['install', requirement])
+
+
+try:
+    from bs import BeautifulSoup
+except ImportError:
+    user_answer = None
```
Comment (pivancy) [inline vasyl_varyshchuk/Week_3/google_search_request.py]: It's odd block of code.

It's enough to create `requirements.txt` file and handle `ImportError`.
In your handler you should write a message like `This script requires `bs` module. Please install it using `requirements.txt` or something similar. That's it.

Comment (vvaryshchuk) [reply vasyl_varyshchuk/Week_3/google_search_request.py]: I realized. I changed handler for ImportError to avoid redundancy in the code.

**Comment by pivancy** [comment]
:+1:

**Pull request merged**